jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errordetails/ExecutionErrorDetailsPresenter.java

```diff
@@ -60,14 +62,17 @@
     @Inject
     private Caller<ExecutorService> executorServices;
 
+    @Inject
+    BreadcrumbManagement breadcrumbs;
```

Comment:
@nmirasch  can we move this to a more generic location (AbstractMultiGridPresenter)? As well as the breadcrumb related methods, that way its easier to maintain any related changes.